lib/pattern-common.ts

```diff
@@ -0,0 +1,46 @@
+import { ValidationError } from 'apollo-server-errors';
```

Comment:
nit: Follow the filename patternt. E.g: patternCommon.ts

Reply:
this is a NPM package. can't change it :/

Reply:
@cabelitos he means our own file (`pattern-common.ts`)

Reply:
changed.